src/distribution/laplace.rs

```diff
@@ -0,0 +1,363 @@
+use distribution::{Continuous, Univariate};
+use rand::distributions::Distribution;
+use rand::distributions::OpenClosed01;
+use rand::Rng;
+use statistics::*;
+use std::f64;
+use {Result, StatsError};
+
+/// Continuous Univariate Laplace distribution.
+/// The Laplace distribution is a distribution over the real numbers parameterized by a mean and
+/// scale parameter. The PDF is:
+///     p(x) = \frac{1}{2 * scale} \exp{- |x - mean| / scale}.
+/// <a href="http://en.wikipedia.org/wiki/Laplace_distribution">Wikipedia - Laplace distribution</a>.
+pub struct Laplace {
+    location: f64,
+    scale: f64,
+}
+
+impl Laplace {
+    fn sample_unchecked<R: Rng + ?Sized>(r: &mut R, location: f64, scale: f64) -> f64 {
+        let r: f64 = r.gen();
+        let u = r - 0.5;
+        location - (scale * f64::signum(u) * f64::ln(1.0 - (2.0 * f64::abs(u))))
+    }
+
+    fn valid_parameter_set(location: f64, scale: f64) -> bool {
```

Comment:
Since the conditional is so simple I think (although it might sound counter-intuitive) that it would almost be more readable just checking it from within `new` rather than pulling it out into it's own function. See [this implementation of new for the normal distribution](https://github.com/boxtown/statrs/blob/master/src/distribution/normal.rs#L48)